### PR TITLE
WRO-6262: Fixed editable scroller to select item by long press

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
UX changed to select item in editable scroller by long pressing the OK button or enter key.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Used `@enact/ui/touchable.hold` to handle long press of the OK button, and long press of the enter key was handled by setTimeout.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-6262

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]

File: Scroller/EditableWrapper.js [hunk 1]
```diff
@@ -37,6 +40,12 @@ const EditableShape = PropTypes.shape({
 
 const SpotlightAccelerator = new Accelerator([5, 4]);
 
+const pressDuration = 500;
```
Comment (jeonghee27) [inline Scroller/EditableWrapper.js:43]:
```suggestion
const holdDuration= 500;
```
Sorry, I don't like variable name

File: Scroller/EditableWrapper.js [hunk 2]
```diff
@@ -37,6 +40,12 @@ const EditableShape = PropTypes.shape({
...
+
+const holdConfig = {
+	events: [{time: pressDuration}]
+};
```
Comment (jeonghee27) [inline Scroller/EditableWrapper.js:47]:
```suggestion
const holdConfig = {
	events: [
		{name: 'hold', time: pressDuration}
	]
};
```

Comment (0x64) [reply Scroller/EditableWrapper.js:47]: @jeonghee27 is right. I missed that `name` becomes `undefined` if we don't specify in `holdConfig`.

File: Scroller/EditableWrapper.js [hunk 3]
```diff
@@ -89,6 +97,13 @@ const EditableWrapper = (props) => {
 
 		// Last InputType which moves Items
 		lastInputType: null,
+
+		// Timer for holding key input
+		timer: null,
```
Comment (jeonghee27) [inline Scroller/EditableWrapper.js:102]:
```suggestion
		keyHoldTimer: null,
```
I'm not sure... 
Please more specific name.

Comment (juwonjeong) [reply Scroller/EditableWrapper.js:102]: @jeonghee27 How about `keyHoldTimerId`?
and `needToPropagateEvent` instead of `stopPropagationFlag` ?

Comment (jeonghee27) [reply Scroller/EditableWrapper.js:102]: OK . `keyHoldTimerId`
`needToPropagateEvent` better I think

File: Scroller/EditableWrapper.js (hunk 1 above)
Comment (juwonjeong) [reply Scroller/EditableWrapper.js]: It's fixed.

File: Scroller/EditableWrapper.js (hunk 2 above)
Comment (juwonjeong) [reply Scroller/EditableWrapper.js]: It's fixed.

File: Scroller/EditableWrapper.js (hunk 3 above)
Comment (juwonjeong) [reply Scroller/EditableWrapper.js]: It's fixed.

**Pull request merged**